samples/CentOS/docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  aria:
+    #Make sure to provide the right value for image field below and uncomment the below line
+    #image: vishwanathj/aria_011:7
```

Comment:
If we plan to distribute a docker image, perhaps that name should go here by default?

Reply:
Distributing docker images might be a future plan. It occurred  to me as well that this docker-compose file should be updated when we start publishing docker images to docker repo.

Reply:
I haven't seen any more comments on this PR so I think we should be able to merge it in.